### PR TITLE
update cli docs

### PR DESCRIPTION
**Pull request by michaeldjeffrey**
More documentation for https://github.com/helium/router/issues/522 and https://github.com/helium/router/issues/427

### CODE REVIEW

File: docs/Device.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Info
+
+```sh
+router device --id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx [--stop]
+```
+
+Simple information for a device. `<id>` is the device ID from Console, not the `app_eui` or `dev_eui`.
+
+### Tracing
```
Comment (dpezely) [inline docs/Device.md:31]: Is tracing part of the info command?  If a separate command, there's no usage/example.  If part of the info command as implied by `--stop` mentioned below, it's confusing being within a separate subsection.

File: docs/Organizations.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Organizations CLI
+
+Router only cares about organizations from Console insofar as it can use DC for a device.
+
+Similar to Devices, Console is the source of truth for Organizations DC balance.
```
Comment (dpezely) [inline docs/Organizations.md:5]: Keeping consistent with usage farther below:

```suggestion
Similar to Devices, Console is the source of truth for an organization's DC balance.
```

File: docs/Organizations.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Single listing of `dc_balance` and `nonce` for `<org_id>`.
+
+### Refill
+
+```sh
+router organization update <org_id> -b <balance> [--commit]
+```
+
+A way to update an organizations DC balance in Console from Router.
```
Comment (dpezely) [inline docs/Organizations.md:37]:
```suggestion
A way to update an organization's DC balance in Console from Router.
```

File: docs/Device.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+# Device CLI Commands
+
+Router maintains a databse in Rocks of Devices, but the paired Console instance should always be viewed as the source of truth for a Device.
```
Comment (dpezely) [inline docs/Device.md:3]:
```suggestion
Router maintains a database in RocksDB of Devices, but the paired Console instance should always be viewed as the source of truth for a Device.
```

**Review by dpezely — CHANGES_REQUESTED** [review]
Mostly minor wording nitpicks, plus one spot where wording wasn't clear on first reading.

**Review by dpezely — DISMISSED** [review]

**Review by macpie — APPROVED** [review]

**Pull request merged**